pkg/gds/service.go

```diff
@@ -138,7 +138,7 @@ func (s *Service) Serve() (err error) {
 		go s.CertManager()
 
 		// Start the backup manager go routine process
-		go s.BackupManager()
+		go s.BackupManager(nil, nil)
```

Comment:
```suggestion
		go s.BackupManager(nil)
```

Reply:
I don't know if it's possible to accept my suggestions and have a working PR - but I've added this one in for completeness just in case - feel free to ignore.